### DISCUSSION TITLE
Any way to update a mongodb document from a Keda MongoDB trigger spec?

### DISCUSSION THREAD

**Discussion by n2nco** [kedacore/keda-docs#911] [Q&A]
For example: when a document meets a trigger threshold, update the document to reflect 'triggered'

https://keda.sh/docs/2.8/scalers/mongodb/

If not, any other recommendation on doing this in the simplest/cleanest way?

**Comment by tomkerkhove** [comment]
No, we do not change the data plane.

But our upcoming CloudEvent support should be able to help you.
